marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx

```diff
@@ -25,6 +25,10 @@ describe('json scenario for snippet on browse documents page', () => {
     browsePage.waitForTableToLoad();
   });
 
+  afterEach(() => {
+    cy.resetTestUser();
+  })
+
```

Comment:
I added this since we are running tests by setting hub-central-developer role to hc-test-user.